### DISCUSSION TITLE
Uploading avatar to Vroid Hub

### DISCUSSION THREAD

**Discussion by tomtomtong** [pixiv/vroid-sdk-developers#35] [Q&A]
Hello. Would I upload my 3D model to Vroid Hub by using the SDK?

**Comment by Haresaku** [comment]
We have developed an SDK for uploading models to VRoid Hub, which is used in an application called MakeAvatar, for example.
https://hub.vroid.com/apps/a1d369bd9528c800e009c071f087437cfc3c28e767f28f8fbcb964e8adf03ef
However, since there are many elements under consideration, we ask for details of the application, its usage and scale before offering it.
We would appreciate it if you could apply to the inquiry form at the link below.
https://www.pixiv.net/support?mode=inquiry&type=43&service=vroid-sdk

**Reply by tomtomtong** [reply]
Thank you for your reply. I just apply for latest Vroid SDK for uploading avatars. My application is this one. https://booth.pm/en/items/3683370
